### PR TITLE
Week 3, Step 1

### PR DESCRIPTION
**Pull request by eldrickb**
Added my name to a file (was told to PR here).

I created a new folder with the original unedited portfolio (off the main repo) because I ran into conflicts with using both NPM and Maven in the same project, I'll most likely ping you about this tomorrow

Thanks

### CODE REVIEW

File: server/pom.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <plugins>
+      <!-- Provides `mvn package appengine:run` for local testing
+           and `mvn package appengine:deploy` for deploying. -->
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>appengine-maven-plugin</artifactId>
+        <version>2.2.0</version>
+        <configuration>
+          <!-- TODO: set project ID. -->
+          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
```
Comment (mmallett) [inline server/pom.xml:38]: Set your project id here

File: server/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@WebServlet("/data")
+public class DataServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("text/html;");
+    response.getWriter().println("<h1>Hello Eldrick!</h1>");
```
Comment (mmallett) [inline server/src/main/java/com/google/sps/servlets/DataServlet.java:30]: This is where you put in your name right? The rest is a fork of the main repo?

**Comment by mmallett** [comment]
Please be sure to add @win-chen as a collaborator and reviewer as well

File: server/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (eldrickb) [reply server/src/main/java/com/google/sps/servlets/DataServlet.java:30]: Yup

**Pull request merged**